sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -200,7 +200,10 @@ def _wrap_eventdata(
             wrapper_event_data._message = trace_message(  # pylint: disable=protected-access
                 wrapper_event_data._message,  # pylint: disable=protected-access
                 amqp_transport=self._amqp_transport,
-                parent_span=span
+                additional_attributes={
+                    TraceAttributes.TRACE_NET_PEER_NAME_ATTRIBUTE: self._client._address.hostname,  # pylint: disable=protected-access
+                    TraceAttributes.TRACE_MESSAGING_DESTINATION_ATTRIBUTE: self._client._address.path,  # pylint: disable=protected-access
```

Comment:
is it an entity path or name? I.e. on the consumer, does it contain `/partition/{id}`  in addition to topic name?
It should be the name only.

Reply:
This corresponds to eventhub name and does not the contain the partition portion.

e.g.:

```
    "attributes": {
        "az.namespace": "Microsoft.EventHub",
        "messaging.system": "eventhubs",
        "messaging.operation": "receive",
        "messaging.batch.message_count": 5,
        "net.peer.name": "paulvaneck-test.servicebus.windows.net",
        "messaging.destination.name": "/eh-paulvaneck-hub"
    },
```

However, it does have the starting slash. Is that fine?